dev/wlp-gradle/subprojects/fat.gradle

```diff
@@ -48,22 +48,22 @@ dependencies {
   derby 'org.apache.derby:derby:10.11.1.1'
 
   jdbcDrivers 'com.ibm.db2:jcc:11.1.4.4',
-		'org.postgresql:postgresql:42.2.5',
-		'com.microsoft.sqlserver:mssql-jdbc:9.2.1.jre8',
-		'org.apache.derby:derbyclient:10.11.1.1',
-		'org.apache.derby:derby:10.11.1.1',
-		'com.oracle.database.jdbc.debug:ojdbc8_g:21.1.0.0'
+    'org.postgresql:postgresql:42.2.5',
+    'com.microsoft.sqlserver:mssql-jdbc:9.2.1.jre8',
+    'org.apache.derby:derbyclient:10.11.1.1',
+    'org.apache.derby:derby:10.11.1.1',
+    'com.oracle.database.jdbc.debug:ojdbc8_g:21.1.0.0'
 
   jakartaTransformer 'biz.aQute.bnd:biz.aQute.bnd.transform:5.3.0',
```

Comment:
why didn't we update the existing wrapper project to use the latest version? - and then keep the "project" dependency